docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -1,168 +1,181 @@
 Enable your users to receive mobile push messages sent from the Apple (APNs) and Google (FCM/GCM) platforms. The Amplify CLI deploys your push notification backend using [Amazon Pinpoint](http://docs.aws.amazon.com/pinpoint/latest/developerguide/).
 You can also create Amazon Pinpoint campaigns that tie user behavior to push or other forms of messaging.
 
-## Pre-requisite
+## Set Up Your Backend
+
+### Prerequisites
 
 - Enrolled in the Apple Developer Program
 - Actual device to run the app on
 
+### Step-by-step
+
 1. Complete the [Get Started](~/start/start.md) steps before you proceed.
 
-2. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
+1. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
```

Comment:
Is this in Xcode? Maybe be a bit more specific where "under the target" is

Reply:
Good call